qa/TL3_RN50_convergence/test_tensorflow.sh

```diff
@@ -38,16 +39,15 @@ if [[ $RET -ne 0 ]]; then
     CLEAN_AND_EXIT 2
 fi
 
-MIN_TOP1=75.0
-MIN_TOP5=92.0
-MIN_PERF=9500
 
-TOP1=$(grep "^Top-1" $LOG | awk '{print $3}')
-TOP5=$(grep "^Top-5" $LOG | awk '{print $3}')
+MIN_TOP1=0.75
+MIN_TOP5=0.92
+MIN_PERF=8000
```

Comment:
why did we lower the threshold?

Reply:
1) TF2.0 seems to be slower
2) It is RN50 v1.5 which is a bit slower but more accurate AFAIK.
3) It consumes more memory and we need to run smaller batch which lowers the perf as well.